resources/i18n/en.json

```diff
@@ -6,7 +6,7 @@
     "name":"English",
     "error-title":"Error",
     "error-downloader-window-title":"Cannot create downloader",
-    "error-downloader-launch-message":"Impossible to launch downloader, Kiwix-desktop will start but all download functions will not working!",
+    "error-downloader-launch-message":"Impossible to launch downloader, Kiwix will start but all download functions will not working!",
```

Comment:
Please also fix the grammar in this message. It should rather read

"Impossible to launch the downloader, Kiwix will start but download functionality will not work!"